proof-producer/libs/assigner/src/trace_parser.cpp

```diff
@@ -0,0 +1,18 @@
+#include <nil/proof-generator/assigner/trace_parser.hpp>
+
+namespace nil {
+    namespace proof_generator {
+
+        std::optional<std::string> trace_parser::get_next_stack_op() {
+            return {};
```

Comment:
Is this code something partial, and we're planning to fill these functions later?

Reply:
yes, it an prototype. The actual implementation will be - https://www.notion.so/nilfoundation/Iterative-EVM-trace-parser-12ec61485260808b9797f023df679542